source/MapOutfitterPanel.cpp

```diff
@@ -243,13 +262,32 @@ void MapOutfitterPanel::Init()
 					catalog[outfit->Category()].push_back(outfit);
 					seen.insert(outfit);
 				}
+
+	// Add outfits in storage at the selected planet or system.
+	for(const auto &it : player.PlanetaryStorage())
```

Comment:
This function seems to be creating the initial list of all Outfits known to the player, why not just add all outfits from storage directly? (Why filter on selected planets and systems here? I don't expect re-initialization every time another planet or system is selected.)

Reply:
Honestly, I think that's just a leftover because I wrote this part of the patch before I wrote the `inStorageInSystem` part. I'll try to reproduce and fix.

Reply:
Reproduced: storing an Ion Cannon on Valhalla and landing on Luna, the Ion Cannon wasn't listed.  I removed the condition in 7408a7ace6b2b14a474912d5f2205b0eb856db71 and now it does get listed, also when I select the yellow `Sells this outfit` filter.

Reply:
Good, please feel free to press the "Mark as resolved" button for this conversation (I don't have permissions to do that).

And one small note; the ES style guide recommends the &-sign attached to the variable and not to the type/keyword.

Reply:
Fixed the ampersand placement and resolving.